example.json

```diff
@@ -15,6 +15,7 @@
     "mobileInstruction": "This is optional instruction text that will be shown when viewed on mobile.",
     "_setCompletionOn":"allItems",
     "_useGraphicsAsPins": false,
+    "_useNumbersAsPins": false,
```

Comment:
Possible attributes for `_pinStyle`:

* **default** - regular pin
   * Also either provides extra attribute or uses the `_classes` to chose what type of pin.
* **graphic** - Graphic implementation
* **numbers** - Numbers pin (this PR)